app/src/main/java/org/p2p/wallet/home/repository/HomeLocalRepository.kt

```diff
@@ -9,6 +9,7 @@ interface HomeLocalRepository {
     suspend fun removeIfExists(publicKey: String, symbol: String)
     fun getTokensFlow(): Flow<List<Token.Active>>
     suspend fun getUserTokens(): List<Token.Active>
+    suspend fun findUserTokenByMint(mintAddress: String): Token.Active?
```

Comment:
use Base58String

Reply:
and could you put this method to UserTokensRepository? 
HomeLocal name doesn't tell us anything about user tokens

Reply:
@gslevinkov I haven't understood your point. HomeLocalRepository already has `getUserTokens` and `getUserBalance` methods. I could rename these methods to `getTokens` and `getBalance` respectively, but would that make sense?